src/WebJobs.Script.WebHost/Metrics/LinuxContainerMetricsPublisher.cs

```diff
@@ -247,6 +249,7 @@ internal async Task SendRequest<T>(ConcurrentQueue<T> activitiesToPublish, strin
         public void Initialize()
         {
             _stampName = _environment.GetEnvironmentVariable(EnvironmentSettingNames.WebSiteHomeStampName);
+            _resourceId = _environment.GetEnvironmentVariable(EnvironmentSettingNames.WebsiteArmResourceId);
```

Comment:
Looks like this WEBSITE_ARM_RESOURCE_ID env var is never set by the platform. So likely we can't rely on this currently.

Reply:
Yes, as discussed, I will raise a PR for the platform in order to populate this variable.